examples/run/alpaka/full_chain_algorithm.hpp

```diff
@@ -122,9 +118,9 @@ class full_chain_algorithm
 
     private:
     /// Alpaka Queue
-    traccc::alpaka::queue m_queue;
+    std::shared_ptr<traccc::alpaka::queue> m_queue;
```

Comment:
Why shared pointer? You explicitly disallow the copy of the class anyway. Then why not a unique pointer?